packages/web3-validator/src/private_types.ts

```diff
@@ -0,0 +1,8 @@
+// To avoid package dependency we have to copy these types
```

Comment:
Should we create a `web3-types` package? There were other instances where packages had dev-dependencies only because they were using types

Reply:
Yes we may need `web3-types` package, but for few lines of duplicate code we should not have a dependency of a package.